src/Avalonia.Base/Avalonia.Base.csproj

```diff
@@ -5,9 +5,6 @@
     <RootNamespace>Avalonia</RootNamespace>
     <AllowUnsafeBlocks>True</AllowUnsafeBlocks>    
   </PropertyGroup>
-  <ItemGroup>
-    <ProjectReference Include="..\Avalonia.Build.Tasks\Avalonia.Build.Tasks.csproj"/>
```

Comment:
It's required to be built by Avalonia.MicroCom and probably other assemblies in the future, so it made sense to reference it from Avalonia.Base.

Reply:
Hi,
> It's required to be built by Avalonia.MicroCom and probably other assemblies in the future, so it made sense to reference it from Avalonia.Base.

from the tests I made Avalonia.MicroCom works correctly.

I don't understand what the purpose of Avalonia.Base dependency on Avalonia.Build.Tasks is. If it is for compiling the xaml of the Core classes, I would add Avalonia.Build.Tasks as PrivateAssets in Base.prop and buildTransitive in Avalonia.nupkg.

Tell me if I have to close this PR:

Reply:
Ah, MicroCom is [also referencing](https://github.com/AvaloniaUI/Avalonia/blob/c85fa2b9977d251a31886c2534613b4730fbaeaf/src/Avalonia.MicroCom/Avalonia.MicroCom.csproj#L10-L13) the build tasks. So it should be OK to remove the reference from Avalonia.Base.